### PR TITLE
Adding small file writes to scale tests

### PR DESCRIPTION
**Pull request by hkadam134**
# Description

Summary: 
* The script manages CephFS operations on a Ceph cluster. It initializes clients and subvolumes, mounts them using FUSE or kernel, and runs smallfile IO operations in parallel. 
* It periodically logs Ceph cluster details and collects MDS logs, which are compressed after testing. 
* The script uses threading to handle concurrent logging and IO operations efficiently.

Logs : 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-X17M8C/
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-J6T4YU/

Ex : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-J6T4YU/sf_w_kernel_10_cl_10_subv_create_ops_1_th_100000_files_1024_size/

This folder has all ceph cluster details like, logs, health status, top etc.

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by sumabai — APPROVED** [review]
lgtm

**Review by sumabai — CHANGES_REQUESTED** [review]
Please resolve tox issues

**Review by neha-gangadhar — COMMENTED** [review]
lgtm

File: tests/cephfs/cephfs_scale/small_file_writes.py [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+                if result:
+                    metrics_outputs.update(result)
+            except Exception as e:
+                log.error(f"Exception occurred during IO operation: {e}")
+
+    log.info(f"Metrics Outputs : {metrics_outputs}")
+    return metrics_outputs
+
+
+def get_daemon_names(client, deamon_type):
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_scale/small_file_writes.py:317]: Do you want to move this to fs utils file. as this can be reused

**Pull request closed without merging**